README.md

```diff
@@ -117,7 +117,7 @@ Parabeac-core keeps track of how many times it is run. Although we do not collec
 5. Save Main.dart and Execute `flutter run` in your terminal
 
 # How to create & run Parabeac eggs
-Parabeac eggs are essentially plugins that change the way a set of design elements are interpreted. A very simple example of this are 2 eggs that we created called NavBar & TabBar. Eggs are loaded into the project before the runtime of Parabeac-Core because unfortunately dart doesn't support [dynamic module loading](https://github.com/dart-lang/sdk/issues/10530).
+Parabeac eggs are essentially plugins that change the way a set of design elements are interpreted. A very simple example of this is 2 eggs that we created called NavBar & TabBar. Eggs are loaded into the project before the runtime of Parabeac-Core because unfortunately dart doesn't support [dynamic module loading](https://github.com/dart-lang/sdk/issues/10530).
```

Comment:
Rather than just changing are to is, I think the correct way of fixing this would be to change the whole sentence to: "Two very simple examples of this are the Navbar and TabBar eggs.".